src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.cs

```diff
@@ -415,7 +416,7 @@ internal string GetRealAnalyzerLoadPath(string originalFullPath)
                             return resolvedAssembly;
                         }
                     }
-                    catch
+                    catch (Exception ex) when (FatalError.ReportAndCatch(ex, ErrorSeverity.Diagnostic))
```

Comment:
@jjonescz your comment reminded me that we're essentially silently swallowing the exceptions from plugins. Changed that to be non-silent. It's not a fatal error but definitely something that should at least raise an alert if it happens. This would've caught the issue that I fixed with this PR.